softwarereview_editor.Rmd

```diff
@@ -142,7 +142,7 @@ Each submission should be reviewed by _two_ package reviewers. Although it is fi
 
 ### Package promotion:
 
--  Ask authors to write either a blog post or a tech-notes post for the package, as appropriate, and ping [Stefanie Butland](https://github.com/stefaniebutland), rOpenSci community manager.
+-  Ask authors to write either a blog post or a tech-notes post for the package, as appropriate, and ping [`@ropensci/blog-editors`](https://github.com/orgs/ropensci/teams/blog-editors), rOpenSci blog editors.
```

Comment:
This sounds like a repeat of 
> Nominate a package to be featured in an rOpenSci blog post or tech note if you think it might be of high interest. Please note in the software review issue one or two things the author could highlight, ...

from section 8.2.4.

and I prefer the language of 8.2.4 suggesting what to highlight